### ISSUE TITLE
check vector version

### ISSUE THREAD

**Issue by patsevanton** [buluma/ansible-role-vector#3]
## Describe the bug

No such file or directory: b''vector'''

## Playbook

```yaml
  - hosts: vector
    become: true
    roles:
      - buluma.vector
```

## Inventory
```
all:
  children:
    vector:
      hosts:
        "vector":
          ansible_host: "xxxx"
  vars:
    ansible_user:  debian
    ansible_ssh_private_key_file: ~/.ssh/id_rsa
```

## Output

```
TASK [ansible-role-vector : check vector version] ****************************************************************************************************************************************
Sunday 10 April 2022  11:07:08 +0600 (0:00:00.882)       0:00:05.232 **********
fatal: [vector]: FAILED! => changed=false
  cmd: vector --version
  msg: '[Errno 2] No such file or directory: b''vector'''
  rc: 2
  stderr: ''
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
...ignoring

```

## Environment
Version
```
ansible [core 2.12.2]
  python version = 3.8.10 (default, Nov 26 2021, 20:14:08) [GCC 9.3.0]
```

Server where ansible
```
cat /etc/*release*
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.3 LTS"
NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
```

Remote server
```
Debian 11
```

**Comment by patsevanton** [comment]
Fix by https://github.com/buluma/ansible-role-vector/pull/2

**Comment by buluma** [comment]
thanks mate, available in [v0.1.3](https://galaxy.ansible.com/buluma/vector)
